themes/bootstrap3/js/account_ajax.js

```diff
@@ -166,17 +166,25 @@ $(document).ready(function registerAccountAjax() {
     selector: ".checkedout-status",
     ajaxMethod: "getUserTransactions",
     render: function render($element, status, ICON_LEVELS) {
+      function titleCase(str) {
```

Comment:
I think it's probably safer to address this through the language files rather than doing it on the fly; this may not be appropriate for all languages.

Reply:
I agree. I didn't want to mess with the language files too much and we're also borrowing strings from other contexts and I didn't want to cause Random Capitalization Elsewhere In The Interface.